scripts/globals/effects/berserk.lua

```diff
@@ -7,16 +7,16 @@ require("scripts/globals/status")
 -----------------------------------
 
 function onEffectGain(target,effect)
-    target:addMod(dsp.mod.ATTP,25)
-    target:addMod(dsp.mod.RATTP, 25)
-    target:addMod(dsp.mod.DEFP,-25)
+    target:addMod(dsp.mod.ATTP, 25 + effect:getPower())
+    target:addMod(dsp.mod.RATTP, 25 + effect:getPower())
+    target:addMod(dsp.mod.DEFP, -25 - effect:getPower())
```

Comment:
yeeeahh lets put the 25 up in the ability script and then you can just put effect:getpower() to a variable in this script and +/- the variable in all these lines.

Reply:
Will do.